solutions/usaco-968.mdx

```diff
@@ -2,51 +2,38 @@
 id: usaco-968
 source: USACO Silver 2019 December
 title: Milk Visits
-author: Qi Wang, Tanish Tyagi
+author: Qi Wang, Tanish Tyagi, Kevin Sheng
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
+[Official Analysis (c++)](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
 
 **Time Complexity:** $\mathcal{O}(N + M)$
 
-We can use a DFS to create connected components from an adjacency list (which is
-made when we process the roads between the farms). Each connected component will
-contain farms that all have the same breed.
+We perform a search on the tree, identifying connected components of cows that
+all have the same breed.
 
-Adjacency List based off sample test case:
+In the sample test case, there's $3$ connected components:
```

Comment:
```suggestion
In the sample test case, there are $3$ connected components:
```

Reply:
uh could you elaborate? both seem non-awkward tbh

Reply:
the first one is grammatically wrong (i think)